sputnikdao-factory2/src/factory_manager.rs

```diff
@@ -7,13 +7,13 @@ use near_sdk::serde_json;
 use near_sdk::{env, sys, AccountId, Balance, CryptoHash, Gas};
 
 /// Gas spent on the call & account creation.
-const CREATE_CALL_GAS: Gas = Gas(75_000_000_000_000);
```

Comment:
This was absurd. A simple `new` call to create a DAO does not require more than `5_000_000_000_000` gas. However, I've doubled it to `10_000_000_000_000` just to be safe.